client/app/visualizations/map/index.js

```diff
@@ -308,6 +340,20 @@ const MapEditor = {
 
     this.mapTiles = MAP_TILES;
 
+    this.iconShapes = {
+      marker: 'Marker',
+      circle: 'Circle',
+      rectangle: 'Rectangle',
+      'circle-dot': 'Circle Dot',
+      'rectangle-dot': 'Rectangle Dot',
+      doughnut: 'Doughnut',
+    };
```

Comment:
I know these are the original labels, but they're not very accurately descriptive.

@deecay How do you feel about these?
```js
{
  marker: 'Marker + Icon',
  doughnut: 'Circle',
  'circle-dot': 'Circle Dot',
  circle: 'Circle + Icon',
  'rectangle-dot': 'Rectangle Dot',
  rectangle: 'Rectangle + Icon',
}
```
<img width="525" alt="Screen Shot 2019-06-15 at 17 43 30" src="https://user-images.githubusercontent.com/486954/59552886-40b4ca80-8f95-11e9-9077-202ef6a99309.png">

Reply:
I think it's now clearer. The remaining thing is if we want "Rectangle" or "Square".

Because:
* `Rectangle Dot` is actually square dot (you can see that in your screenshot's orange square)
* As discussed, I would like to make `Rectangle` actually a square.